spec/request/service_instances_spec.rb

```diff
@@ -3495,51 +3774,6 @@ def enable_sharing!
     let!(:service_instance2) { VCAP::CloudController::ManagedServiceInstance.make(space: space, name: 'redis') }
     let!(:service_instance3) { VCAP::CloudController::ManagedServiceInstance.make(space: another_space, name: 'mysql') }
 
-    describe 'POST /v3/service_instances/:guid/relationships/shared_spaces' do
```

Comment:
I guess we want to delete the entire `unrefactored` describe that this one is in?

Reply:
still have delete tests to finish  so will remove unrefactored and the controller unit test after that

Reply:
I have removed all POST related tests